.travis.yml

```diff
@@ -84,7 +84,7 @@ script:
       ./node_modules/.bin/grunt flow
 
       ALL_FILES=`find src -name '*.js' | grep -v umd/ | grep -v __tests__ | grep -v __mocks__`
-      COUNT_ALL_FILES=`echo $ALL_FILES | wc -l`
```

Comment:
`echo "$ALL_FILES"` should work. I'd prefer that over `ls` as it's clearer intent.